DatadogCore/Sources/Datadog.swift

```diff
@@ -301,6 +301,17 @@ public enum Datadog {
         core?.clearAllData()
     }
 
+    /// Stops the initialized SDK instance attached to the given name.
+    ///
+    /// Stopping a core instance will stop all current processes by deallocating all Features registered
+    /// in the core as well as their storage & upload units.
+    /// 
+    /// - Parameter instanceName: the name of the instance to stop.
+    public static func stopInstance(named instanceName: String = CoreRegistry.defaultInstanceName) {
+        let core = CoreRegistry.unregisterInstance(named: instanceName) as? DatadogCore
+        core?.stop()
```

Comment:
question/ Why not making `stop()` a requirement in `DatadogCoreProtocol`? I have no opinion on this (both may be fine), just wondering on what was the rationale.

Reply:
The stop is only required from high level apis, there is no usage from Features so there no need for making it part of the protocol. The requirement might change in the future tho!